### PR TITLE
Testing docs build

### PR DESCRIPTION
**Pull request by hoxbro**
Reverts holoviz/panel#6833

Docs passes: https://github.com/holoviz/panel/actions/runs/9114403040

### CODE REVIEW

**Comment by maximlt** [comment]
I think the fix is to simply remove the `.html` suffix in both keys and values. cc @Coderambling

**Comment by Coderambling** [comment]
D*rn I had not realized that.

Is it still OK to do i.e.:

 'user_guide/APIs : 'explanation/api/index.html#apis' ? As that 2nd part (the value) does not end in .html? Or would that also cause issues?

**Comment by hoxbro** [comment]
Have set over another run here: https://github.com/holoviz/panel/actions/runs/9130979122

File: doc/conf.py [hunk 1]
```diff
@@ -97,14 +97,11 @@
 rediraffe_redirects = {
     # Removal of the developer testing page
     'developer_guide/testing': 'developer_guide/index',
-    'user_guide/APIs': 'explanation/api/index.html#apis',
+    'user_guide/APIs': 'explanation/api/index#apis',
     'user_guide/Pipelines': 'how_to/pipeline/index',
     'user_guide/Templates': 'how_to/templates/index',
     '_static/images/sazure_deployment.png': '_static/images/azure_deployment.png',
```
Comment (philippjfr) [inline doc/conf.py:103]: Pngs can't be redirected it seems

```suggestion
```

**Pull request merged**